### DISCUSSION TITLE
Disable `d` operator when remapped to a regular motion

### DISCUSSION THREAD

**Discussion by axonasif** [VSCodeVim/Vim#7700] [Q&A]
Hi! I have this config on my `settings.js`. The scrolling up part works fine. However, when `d` is pressed, it stays hung for a second on `operatorPendingMode` and then scrolls down. Is there anything I could do to completely disable the `operatorPendingMode` for `d` keyPress and make it act like a regular motion? I remapped the delete operator to `C-d`, which is working as intended.

```json
	"vim.normalModeKeyBindingsNonRecursive": [
		{
			"before": [
				"u"
			],
			"after": [
				"<C-u>"
			],
		},
		{
			"before": [
				"<C-u>"
			],
			"after": [
				"u"
			]
		},
		{
			"before": [
				"d"
			],
			"after": [
				"<C-d>"
			],
		},
		{
			"before": [
				"<C-d>"
			],
			"after": [
				"d"
			],
		},
	],
```

**Comment by ttytm** [comment]
Hey @axonasif ,

you'll probably also need to care about `d` in `"vim.operatorPendingModeKeyBindings"`

**Reply by axonasif** [reply]
oh, gotta see. Thanks.
